src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,13 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+  const firstTwoYears = 15 + 9;
 
-module.exports = humanCatDogYears;
+  const animalYears = function (catOrDogMultiplier) {
+    return catOrDogMultiplier * (number - 2) + firstTwoYears;
+  };
+
+  const catYears = animalYears(4);
+  const dogYears = animalYears(5);
+  return Array.of(number, catYears, dogYears);
+};
```

Comment:
this works if human years is equal to 2 or more. However, you get the incorrect output if human years is 0 or 1.

Reply:
the simplest way to fix this it just to add 2 `if` statements at the top of your function, before `const firstTwoYears = ...`, e.g.
```
if (number === 0) return [0, 0, 0];
if (number === 1) return [1, 15, 15];
```